openmicroscopy/openmicroscopy
# Path display

Display the location of the image.
To test 
- Select a single image. Click on the path icon (see screenshot)
  - One path should be displayed in a small modal dialog.
- Select a multi-image files. Click on the path icon
  - Several paths should be displayed in a small modal dialog. (see screenshot)
    ![singleImage](https://f.cloud.github.com/assets/1022396/178119/901a51da-7bab-11e2-86ff-9c733ae231b2.png)
    ![multi-images](https://f.cloud.github.com/assets/1022396/178123/935b7496-7bab-11e2-903d-6c37de7c9365.png)
